website/docs/d/privatelink_endpoint_service_data_federation_online_archive.markdown

```diff
@@ -46,6 +46,8 @@ In addition to all arguments above, the following attributes are exported:
 * `type` - Human-readable label that identifies the resource type associated with this private endpoint.
```

Comment:
I saw the renaming of *.markdown to *.html.markdown for the resource doc file, could we do the same for the data source files? Just for consistency with other doc files